### ISSUE TITLE
Relationship between scoring system and sampling method

### ISSUE THREAD

**Issue by plannero** [w3c/silver#553]
I love the idea of WCAG3 being able to score outcomes rather than just fail/pass criteria. In theory, this could help focus initial resources on fixing the most urgent problems instead of relatively insignificant glitches. 

But I also believe this - at least with the currently published draft scoring system - would make auditing significantly more time consuming. Possibly to the extent that not many organizations will not be able to perform a manual audit.

Take, for example the rating scale for "Text alternative available" (https://www.w3.org/TR/wcag-3.0/#text-alternatives ): In order to come up with a rating for this single outcome, an auditor would need to find out what percentage "of all images have appropriate text alternatives". For a large website, this could easily mean manual evaluation of thousands of text descriptions of images and other non-text content, as opposed to the current WCAG2 situation where it is enough to point out the most urgent problems, fail the SC and educate content creators about the importance of text alternatives. 

I guess the same risk applies to several other outcome ratings as well.

However, with a good sampling method, the number of text descriptions that must be evaluated can be reduced. Therefore, the formulation of sampling methodology should be coordinated with the formulation of rating/scoring system. 

Well, maybe this is obvious to all involved. If so, feel free to close this issue!

**Comment by cwadamsoforacle** [comment]
Proposed response:  

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines (WCAG) 3.0](https://www.w3.org/TR/wcag-3.0/).  We believe that these issues are no longer relevant based on the changes made to the latest working draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
